### ISSUE TITLE
Quality of life codebase refactoring

### ISSUE THREAD

**Issue by zouharvi** [AppraiseDev/Appraise#134]
For newcomers like me it's difficult to navigate and contribute to the codebase because of the accumulated technical debt. Some examples:
- `TODO` file from 2018
- Misc. scripts in top-level (`install-psql.sh`, `validate.sh`)
- `EvalView/views.py` having 2.5k lines
- Hardcoded [workarounds for WMT22 campaign](https://github.com/AppraiseDev/Appraise/blob/c70f6b900300ddf3a4e2f71271018fa88d91a811/EvalView/views.py#L946).

I would like to open a discussion on cleaning and refactoring. Testing and adding new features would then be much easier.

PS: I don't mean this in a bad way at all. I would like to help out. :slightly_smiling_face:

**Comment by snukky** [comment]
I totally agree, and thank you for considering this! If possible, I'd like to have those changes made in a mainstream branch first (e.g., develop, i.e. without MQM) and merge it to your working branches to avoid a huge branch eventually. So if possible, let's try refactor and open separate PRs towards develop/main.

Afaik, scripts like `install-psql.sh` we use for webapp setup, so let's make sure they are not removed. The `signlt22xxx` are option names, which we make sure we don't drop too during refactoring & cleanup.
